indicators/NO_GLAC_001/R/NO_GLAC_001.qmd

```diff
@@ -0,0 +1,1731 @@
+
+seksjoner <- sf::read_sf(pathD, layer = "Seksjoner2017") |> 
+  as_tibble() |>
+  mutate(Seksjon = case_when(
+    Seksjon_ko %in% c("6SE-4", "6SE-5") ~ "6SE-4-5",
+    .default = Seksjon_ko
+  ))
+
+# unique(seksjoner$Seksjon)
+
+# join based on SSB IDs and convert to sf object again
+BCreg <- dplyr::left_join(soner , seksjoner |> select(-Shape), by = join_by(SSBID)) |>
+  mutate(BCregion = paste(Seksjon, Sone)) |>
+  sf::st_as_sf() |>
+  select(SSBID,
+  Sone,
+  Seksjon,
+  BCregion) |>
+  st_transform(st_crs(greg))
+
+```
+
+```{r makeFig}
+#| eval: false
+#| code-summary: 'Make figure'
+BCreg |>
+  ggplot(aes(x = Sone, fill = BCregion)) +
+  geom_bar() +
+  facet_grid(Seksjon ~ Sone, scales = "free") +
+  guides(fill = "none") +
+  theme(axis.text.x = element_blank(),
+    axis.title.x.bottom = element_blank()) +
+  labs(y = "km-2")
+ggsave("../img/bcreg.png")
+```
+
+```{r fig-sonerOgSeksjoner}
+#| fig-cap: "Distribution of area within each combination of bioclimatic sone and section (i.e. each bioclimatic region)."
+
+knitr::include_graphics("../img/bcreg.png")
+```
+
+```{r}
+#| eval: false
+#| code-fold: true
+#| code-summary: "Union and cache bioclimatic region data"
+
+BCreg <- BCreg |>
+  group_by(BCregion) |>
+  summarize(Shape = st_union(Shape))
+saveRDS(BCreg, paste0(path, "/data/bioclimatic_regions.rds"))
+```
+
+```{r}
+#| include: false
+# read cache
+BCreg <- readRDS(paste0(path, "/data/bioclimatic_regions.rds"))
+```
+
+
+
+### 7.5 Dataset E - Alpine ecosystem extent
+
+I use a map of alpine extent that was also used by @framstad2021. It is
+based on certain open, terrestrial land cover types above the current
+/realised forest line. There is ongoing work to replace the ecosystem
+mask for the alpine main ecosystem. When such a map is ready, the
+workflow here can easily be updated.
+
+```{r}
+#| eval: false
+#| code-fold: true
+#| code-summary: "Prepare and cache alpine extent map"
+
+pathE <- paste0(path, "/data/temp/alpine_extent.tif")
+
+alpine <- stars::read_stars(pathE) |>
+  st_as_sf(merge = T)
+
+# intersections are much quicker on multipolygons, so I will union the geometries. 
+alpine <- st_union(alpine)
+
+saveRDS(alpine, paste0(path, "/data/alpine_extent.rds"))
+```
+
+```{r}
+#| include: false
+alpine <- readRDS(paste0(path, "/data/alpine_extent.rds"))
+```
+
+## 8. Spatial units
+
+I use basically three spatial units: EAs (level 1; 70 units),
+geographical region (level 2, 5 units), and the EAA.
+
+### 8.1 Ecosystem assets (EA)
+
+Here I create a dataset for the EAs. The EAs will be defined as the
+intersection between bioclimatic regions and geographical regions. Then
+I add total mountain area and glacier area as variables to this dataset.
+
+Note that the EA are not contiguous, and so metric values shold not be 
+presented at the EA level as maps since data from other geographical areas inside 
+each EA strata is informed from the entire EA.
+
+```{r}
+#| eval: false
+#| code-summary: 'Create EA dataset'
```

Comment:
I like that you name these code chunks. Makes it really easy to follow the workflow.